### PR TITLE
attachment_zen

### PR DESCRIPTION
**Pull request by grosser**
how about we get a real version released ?

@pschambacher @osheroff @marcosvm @dragonfax 

### Risks
 - None

### CODE REVIEW

**Comment by marcosvm** [comment]
:+1:

File: attachment_fu.gemspec [hunk 1]
```diff
@@ -1,8 +1,8 @@
-Gem::Specification.new "attachment_fu", "1.0.0" do |s|
+Gem::Specification.new "attachment_zen", "1.0.0" do |s|
   s.authors = ["Technoweenie"]
   s.description = "Adds has_attachment (file store) properties to ActiveRecord.  Supports local file, file-in-db, S3 and Cloudfiles backends."
   s.files = `git ls-files lib vendor README.md CHANGELOG`.split("\n")
-  s.homepage = %q{https://github.com/zendesk/attachment_fu}
+  s.homepage = "https://github.com/zendesk/attachment_fu"
```
Comment (pschambacher) [inline attachment_fu.gemspec]: Should we rename the repo as well?

**Comment by pschambacher** [comment]
:+1:

**Comment by grosser** [comment]
I'd rather keep the repo since we are using it directly in a few places

**Pull request merged**